Regions/Ausbaxter_Load region render matrix slot 1.lua

```diff
@@ -0,0 +1,128 @@
+--[[
+ * ReaScript Name: Load region render matrix slot n
+ * Description: Allows quick storage and recall of reaper render matrix.
+ * Instructions: Make track selections in region matrix, use the "complementary Save region render matrix slot n" to save the setup. 
+      Use the 'load' scripts to recall.
```

Comment:
You seem to be using a header style from old scripts. While reapack-index does understand some of these tags, most are discarded or won't behave as intended.

Please refer to the [packaging documentation](https://github.com/cfillion/reapack-index/wiki/Packaging-Documentation) and [examples](https://github.com/cfillion/reapack-index/wiki/Examples) pages for proper headers.

- ReaScript Name becomes Description
- Description and Instructions becomes About
- Author URI, Repository URL, File URI etc becomes Links
- Changelog syntax is different (no longer a separate block)